components/Product/index.jsx

```diff
@@ -0,0 +1,39 @@
+import Link from 'next/link'
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { Button, CardActionArea, CardActions } from '@mui/material';
```

Comment:
For simplicity, you can use `import { Card, Typography, CardMedia, CardContent } from "@mui/material";` instead of multiple imports